src/EditorFeatures/CSharpTest/Completion/CompletionProviders/SymbolCompletionProviderTests.cs

```diff
@@ -11293,5 +11293,37 @@ int M()
     }
 }", "y");
         }
+
+        [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
+        [WorkItem(53930, "https://github.com/dotnet/roslyn/issues/53930")]
+        public async Task TestTypeParameterConstraintedToInterfaceWithStatics()
+        {
+            var source = @"
+interface I1
+{
+
+    static abstract void M1();
```

Comment:
>static abstract void M1(); [](http://example.com/codeflow?start=4&length=26)

Please add a non-abstract static method. It is not valid to offer it in this context.